client/dead.lua

```diff
@@ -94,6 +94,18 @@ local function DrawTxt(x, y, width, height, scale, text, r, g, b, a, _)
     DrawText(x - width/2, y - height/2 + 0.005)
 end
 
+function runRespawn(type)
+    if type == 'vanilla' then
+        return exports['spawnmanager']:spawnPlayer()
```

Comment:
This method is deprecated. I'm looking for another way to handle this.

Reply:
@charj did you work this out?

If not shall we remove this option or handle it differently?

Reply:
I haven't. 
Honestly, I'm not sure what else to use. Have googled very lightly @tom-osborne 

Would appreciate some help with it

Reply:
The issue now with this is that it will respawn you as default hipster skater character
It should respect `qb-multicharacter`